synapse-modules/alkemio_room_control.py

```
@@ -34,77 +34,160 @@
+                    logger.debug(
+                        "Filtered %d hidden rooms from /sync for %s",
+                        len(hidden_room_ids), user_id,
+                    )
+
+
+                return result_builder
+
+            sync_handler.get_sync_result_builder = patched_get_sync_result_builder
+            logger.info("SyncHandler patched for io.alkemio.visibility filtering")
+
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)